From 74747b7e529f03c610f288ddb7952e20329a7102 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Morten=20Johan=20S=C3=B8rvig?=
 <morten.sorvig@theqtcompany.com>
Date: Tue, 1 Sep 2015 15:52:39 +0200
Subject: [PATCH] Platformsupport: Don't include dbus on darwin.

This created a false dependency from the cocoa platform
plugin to QtDBus, which caused macdeployqt to deploy
it.

This change is for libplatformsupport only, and has
no effect on QtDBus usage in general.

Change-Id: I35f342574a6497ff88a785e93eef1acdea8b1452
Task-number: QTBUG-48015
---
 src/platformsupport/platformsupport.pro | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/qtbase/src/platformsupport/platformsupport.pro b/qtbase/src/platformsupport/platformsupport.pro
index 34e2ed3..12a9c5b 100644
--- a/qtbase/src/platformsupport/platformsupport.pro
+++ b/qtbase/src/platformsupport/platformsupport.pro
@@ -22,7 +22,11 @@ include(accessibility/accessibility.pri)
 include(linuxaccessibility/linuxaccessibility.pri)
 include(clipboard/clipboard.pri)
 include(platformcompositor/platformcompositor.pri)
-contains(QT_CONFIG, dbus) {
+
+# dbus convenience, but not for darwin: the platform
+# plugins for these platforms do not use dbus and we
+# don't want to create a false dependency.
+!darwin: contains(QT_CONFIG, dbus) {
     include(dbusmenu/dbusmenu.pri)
     include(dbustray/dbustray.pri)
 }
-- 
2.8.2

